components/MashTags/MashTags.tsx

```diff
@@ -0,0 +1,90 @@
+import moment from 'moment';
+
+interface Props {
+  filter: string;
+  createdAt: string;
+  initialDecision: string | undefined;
+}
```

Comment:
Could we just pass in the whole referral (MashReferral object)?

We don't need filter because MashReferral has a property stage